content/get_start/UseCase_Workflows/pollination.md

```diff
@@ -8,99 +8,102 @@ pre: "<b>- </b>"
 
 {{<line_break>}}
 
-#### B. Use Case 2: Application of the PRM measure to a user-model created in Pollination
+#### B. Use Case 2: Application of the PRM measure to a user model through BIM-Pollination workflow
```

Comment:
Is there a reason why "through BIM" is added for Pollination only? 
Also, I am wondering if there is a better way to write it, something like - "Application of the PRM measure to a BIM user model - Pollination Workflow"